ngmix/gaussmom.py

```diff
@@ -48,56 +43,7 @@ def get_result(self):
 
         return self.result
 
-    def _do_coadd_maybe(self, mbobs):
```

Comment:
This is a big API change. Is there a reason we don't want to allow this?

Reply:
I never used this code gaussmom code anywhere, it was just a sort of copy of the moments stuff from metadetect

Reply:
I think there is no purpose to actually doing this coadding here anyway